projects/pt1/python/torch_mlir/jit_ir_importer/build_tools/abstract_interp_lib_gen.py

```diff
@@ -1127,6 +1127,13 @@ def aten〇conv2d〡shape(input: List[int], weight: List[int], bias: Optional[Li
 def aten〇conv_transpose2d〇input〡shape(input: List[int], weight: List[int], bias: Optional[List[int]] = None, stride: List[int] = (1, 1,), padding: List[int] = (0, 0,), output_padding: List[int] = (0, 0,), groups: int = 1, dilation: List[int] = (1, 1,)) -> List[int]:
     return upstream_shape_functions.conv_transpose2d_input(input, weight, bias, stride, padding, output_padding, groups, dilation)
 
+def aten〇conv_tbc〡shape(self: List[int], weight: List[int], bias: List[int], pad: int = 0) -> List[int]:
```

Comment:
You should make a test of this as mentioned here.

https://github.com/llvm/torch-mlir/pull/2735/files#r1453668408

Reply:
ty & willdo!